tests/ballerina-test/src/test/java/org/ballerinalang/test/types/bytetype/BByteArrayValueTest.java

```diff
@@ -222,4 +222,25 @@ public void testByteArrayLiteral() {
         Assert.assertEquals(returns.length, 1);
         assertResult(bytes, returns[0]);
     }
+
+    @Test(description = "Test return byte array")
```

Comment:
Shall we say "Test returned byte array"?

Reply:
Actually this test case for testing byte array returned and their size. So renamed accordingly.